### PR TITLE
Support ranges in /ipsearch

### PR DESCRIPTION
**Pull request by Slayer95**

### CODE REVIEW

File: config/commands.js [hunk 1]
```diff
@@ -243,18 +243,38 @@ var commands = exports.commands = {
...
+		var results = [];
 		this.sendReply("Users with IP " + target + ":");
-		for (var userid in Users.users) {
-			var curUser = Users.users[userid];
-			if (curUser.latestIp === target) {
-				this.sendReply((curUser.connected ? " + " : "-") + " " + curUser.name);
-				atLeastOne = true;
+
+		var isRange;
+		if (target.from(-1) === '*') {
```
Comment (Zarel) [inline config/commands.js]: `.from`? Is that a Sugar.js thing? I haven't heard of it. Convention is to use `slice`.

File: config/commands.js [hunk 2]
```diff
@@ -243,18 +243,38 @@ var commands = exports.commands = {
...
+			isRange = true;
+			target = target.to(-1);
```
Comment (Zarel) [inline config/commands.js]: `.slice(0, -1)` does the same thing as `.to(-1)`.

File: config/commands.js (hunk 1 above)
Comment (Slayer95) [inline config/commands.js]: No problem, I just thought it made sense in conjuction with <code>to</code>.

File: config/commands.js (hunk 2 above)
Comment (Slayer95) [inline config/commands.js]: That's good to know!

**Comment by Slayer95** [comment]
Updated.

**Pull request merged**